### PR TITLE
F keychain storage

### PR DESCRIPTION
**Pull request by Harleymckee**
store local keys using https://github.com/oblador/react-native-keychain instead of AsyncStorage.

the former is fully encrypted in the device.

Also adds API_URL release variable to Ios and android.

### CODE REVIEW

File: template/src/containers/AuthContainer.tsx [hunk 1]
```diff
@@ -26,17 +28,14 @@ export default class AuthContainer extends Container<AuthContainerState> {
...
-
       await this.setState(
         {currentUser: {...attributes.user, token: attributes.access_token}},
-        () => {
-          AsyncStorage.multiSet([
-            ['token', this.state.currentUser.token],
-            ['email', this.state.currentUser.email],
-            ['id', `${this.state.currentUser.id}`],
-          ])
+        async () => {
```
Comment (Dacello) [inline template/src/containers/AuthContainer.tsx]: 👍

**Comment by Harleymckee** [comment]
@Dacello one note, I am going to add the webpack like aliasing next (in another sub pr)

File: template/src/containers/AuthContainer.tsx (hunk 1 above)
Comment (Dacello) [reply template/src/containers/AuthContainer.tsx]: nit: this `setGenericPassword` happens the same for create user and sign in... Might could make it a reusable function

File: template/src/containers/AuthContainer.tsx [hunk 2]
```diff
@@ -116,5 +114,15 @@ export default class AuthContainer extends Container<AuthContainerState> {
       this.setState({isFetching: false})
     }
   }
+
+  getMe = async () => {
+    await this.setState({isLoading: true})
+
+    const results = await get(`${global.apiUrl}`)
+
+    console.log(results)
```
Comment (Dacello) [inline template/src/containers/AuthContainer.tsx]: :zap:

File: template/src/helpers/api.ts [hunk 3]
```diff
@@ -62,15 +51,16 @@ async function post(url, body) {
   return results
 }
 
-async function put(url, body) {
-  const userToken = await AsyncStorage.getItem('token')
+async function put(url: RequestInfo, body: any) {
+  const credentials = await Keychain.getGenericPassword()
```
Comment (Dacello) [inline template/src/helpers/api.ts]: would be nice to have a reusable `getToken` method so we didnt need to remember that its `credentials.password` every time

**Pull request merged**